src/frontend/src/lib/components/dapps/DappPromoBanner.svelte

```diff
@@ -0,0 +1,40 @@
+<script lang="ts">
+	import Img from '$lib/components/ui/Img.svelte';
+	import { i18n } from '$lib/stores/i18n.store';
+	import type { DApp } from '$lib/types/dapp';
+	import { replacePlaceholders } from '$lib/utils/i18n.utils';
+
+	export let dApp: DApp;
+</script>
+
+<article class="relative flex h-64 items-end overflow-hidden rounded-2xl">
+	{#if dApp.screenshots && dApp.screenshots.length > 0}
+		<div class="absolute">
+			<Img src={dApp.screenshots[0]} alt={dApp.name} />
+		</div>
+	{/if}
+	<div class="flex-1 px-4 py-4 backdrop-blur-sm">
+		<div class="flex items-center gap-x-2">
+			<div class="h-12 w-12 rounded-full">
+				<Img src={dApp.logo} alt={dApp.name} />
```

Comment:
nitpick: just sharing in case you like it, but if you use `@const` you can destruct the property.

```
{@const {logo: src} = dApp}

<Img {src} />
```

Again, not a call to action.